### PR TITLE
fix(Browser): Removed a browser callout 

### PR DESCRIPTION
**Pull request by paperclypse**
The callout's no longer timely or maybe relevant

### CODE REVIEW

File: src/i18n/content/kr/docs/more-integrations/terraform/terraform-intro.mdx [hunk 1]
```diff
@@ -67,7 +67,7 @@ translationType: machine
 
        . 대부분의 사용자 키는 접두사 `NRAK-`로 시작합니다.
 
-    3. 당신의 [New Region 지역](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) . 계정 설정 페이지가 `one.newrelic.com` 에 있는 경우 지역은 `US` 이고 계정이 `one.eu.newrelic.com` 에 있는 경우 `EU` }입니다.
+    3. 당신의 [New Region 지역](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) . 계정 설정 페이지가 one.newrelic.com 에 있는 경우 지역은 `US` 이고 계정이 `one.eu.newrelic.com` 에 있는 경우 `EU` }입니다.
```
Comment (ally-sassman) [inline src/i18n/content/kr/docs/more-integrations/terraform/terraform-intro.mdx:70]:
```suggestion
    3. 당신의 [New Region 지역](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) . 계정 설정 페이지가 one.newrelic.com 에 있는 경우 지역은 `US` 이고 계정이 one.eu.newrelic.com 에 있는 경우 `EU` }입니다.
```

File: src/content/docs/more-integrations/terraform/terraform-intro.mdx [hunk 2]
```diff
@@ -64,7 +64,7 @@ With `terraform` all set, configure the New Relic `provider` with the following
 
 1. Your [New Relic Account ID](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id).
 2. Your New Relic <InlinePopover type="userKey" />. Most user keys begin with the prefix `NRAK-`.
-3. Your [New Relic region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account). Your region is `US` if your account settings page is located at `one.newrelic.com`, and `EU` if your account is located at `one.eu.newrelic.com`.
+3. Your [New Relic region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account). Your region is `US` if your account settings page is located at one.newrelic.com, and `EU` if your account is located at one.eu.newrelic.com.
```
Comment (ally-sassman) [inline src/content/docs/more-integrations/terraform/terraform-intro.mdx:67]: @paperclypse What do you think about removing the formatting from the URL as well the region acronym (EU, US, etc.)?

Comment (paperclypse) [reply src/content/docs/more-integrations/terraform/terraform-intro.mdx:67]: How did this terraform doc get into my PR? Did just accidentally drag some other changes into it? @ally-sassman

File: src/i18n/content/kr/docs/browser/new-relic-browser/configuration/view-browser-apps-alert-information.mdx [hunk 3]
```diff
@@ -9,7 +9,7 @@ translationType: machine
 ---
 
 <Callout variant="important">
-  통합된 경험을 제공하기 위해 브라우저, 모바일 및 모니터(합성) 위반을 더 이상 사용하지 않고 이에 상응하는 New Relic으로 대체합니다. `one.newrelic.com` 로 이동하면 알림 세부정보가 자동으로 표시됩니다. 자세한 내용 [은 New Relic](/docs/new-relic-one/use-new-relic-one/new-relic-ai/introduction-new-relic-ai) 및 [2020년 7월 지원 포럼 게시물](https://discuss.newrelic.com/t/new-relic-feature-end-of-life-announcements/105772) 의 응용 인텔리전스 및 경고에 대한 설명서를 참조하십시오.
+  통합된 경험을 제공하기 위해 브라우저, 모바일 및 모니터(합성) 위반을 더 이상 사용하지 않고 이에 상응하는 New Relic으로 대체합니다. one.newrelic.com 로 이동하면 알림 세부정보가 자동으로 표시됩니다. 자세한 내용 [은 New Relic](/docs/new-relic-one/use-new-relic-one/new-relic-ai/introduction-new-relic-ai) 및 [2020년 7월 지원 포럼 게시물](https://discuss.newrelic.com/t/new-relic-feature-end-of-life-announcements/105772) 의 응용 인텔리전스 및 경고에 대한 설명서를 참조하십시오.
```
Comment (ally-sassman) [inline src/i18n/content/kr/docs/browser/new-relic-browser/configuration/view-browser-apps-alert-information.mdx:12]:
```suggestion
  통합된 경험을 제공하기 위해 브라우저, 모바일 및 모니터(합성) 위반을 더 이상 사용하지 않고 이에 상응하는 New Relic으로 대체합니다. `one.newrelic.com` 로 이동하면 알림 세부정보가 자동으로 표시됩니다. 자세한 내용 [은 New Relic](/docs/new-relic-one/use-new-relic-one/new-relic-ai/introduction-new-relic-ai) 및 [2020년 7월 지원 포럼 게시물](https://discuss.newrelic.com/t/new-relic-feature-end-of-life-announcements/105772) 의 응용 인텔리전스 및 경고에 대한 설명서를 참조하십시오.
```

File: src/i18n/content/kr/docs/more-integrations/terraform/terraform-intro.mdx [hunk 4]
```diff
@@ -67,7 +67,7 @@ translationType: machine
 
        . 대부분의 사용자 키는 접두사 `NRAK-`로 시작합니다.
 
-    3. 당신의 [New Region 지역](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) . 계정 설정 페이지가 `one.newrelic.com` 에 있는 경우 지역은 `US` 이고 계정이 `one.eu.newrelic.com` 에 있는 경우 `EU` }입니다.
+    3. 당신의 [New Region 지역](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) . 계정 설정 페이지가 one.newrelic.com 에 있는 경우 지역은 `US` 이고 계정이 one.eu.newrelic.com 에 있는 경우 `EU` }입니다.
```
Comment (ally-sassman) [inline src/i18n/content/kr/docs/more-integrations/terraform/terraform-intro.mdx:70]:
```suggestion
3. 당신의 [New Region 지역](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account) . 계정 설정 페이지가 `one.newrelic.com` 에 있는 경우 지역은 `US` 이고 계정이 `one.eu.newrelic.com` 에 있는 경우 `EU` }입니다.
```

File: src/content/docs/more-integrations/terraform/terraform-intro.mdx (hunk 2 above)
Comment (ally-sassman) [reply src/content/docs/more-integrations/terraform/terraform-intro.mdx:67]: No, that was my mistake! I removed those edits

Comment (ally-sassman) [inline src/content/docs/more-integrations/terraform/terraform-intro.mdx:67]:
```suggestion
3. Your [New Relic region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers#verifying-account). Your region is `US` if your account settings page is located at `one.newrelic.com`, and `EU` if your account is located at `one.eu.newrelic.com`.
```

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**